lib/modules.nix

```diff
@@ -57,6 +63,48 @@ in
       }
       // extraSpecialArgs;
     };
+
+  /**
+    Build a Nixvim package.
+
+    This is a thin wrapper around `<nixvim-configuration>.config.build.package`.
```

Comment:
This line kinda confused me in the context of the rest of the docstring. We are building a package but it says its a thin wrapper around the output of a built package? Or am I reading it wrong?

Reply:
Ah I see. so `...around the ... option` ?

Reply:
Yeah, that sounds good to me.